### DISCUSSION TITLE
Changing controllers makes it stop working

### DISCUSSION THREAD

**Discussion by Somedude4545** [mupen64plus-ae/mupen64plus-ae#973] [Q&A]
I have been having trouble with using different types of controllers on the emulator. When I switch between an Xbox controller and a regular keyboard is difficult task, it takes me a while and pure luke to make them work. Despite me selection the correct profile of the controller everywhere. In the controller debug it shows the app is recognizing perfectly the controls, but in game it doesn't work. Coul use a little help, I enjoyed the emulator even bought the pro version. Another topic, how to import saves from the regular app to the pro app.

**Comment by fzurita** [comment]
See here on how to transfer data: http://www.paulscode.com/forum/index.php?topic=16976.0

See here on making controllers: http://www.paulscode.com/forum/index.php?topic=15812.0
